### ISSUE TITLE
showAchievements() not showing anything

### ISSUE THREAD

**Issue by RaulSangosto** [Abedalkareem/games_services#70]
await gs.GamesServices.showAchievements(); not showing anything but response is "success".
The achievements are working and I can see them in google play it is just the UI of the showAhievements() that it is not working.

**Comment by Abedalkareem** [comment]
Hi!
Can you provide logs? most probably user is not logged in. Make sure to call sign-in before trying to show the leaderboard.

**Comment by RaulSangosto** [comment]
Hello! thanks for the quick response.
I am correctly log in, in fact everything its working, I get awarded the achievements, the score of the leaderboards are getting post. The only thing not working are the UI functions of showLeaderBoards() and showAchievements() I get a success response but the UI doesnt show up.

I dont get any logs more than the success, I dont know if I have to activate anything to see them.

I am going to check if it is because of some incompatibility with other packages because I followed the flutter casual games template and I noticed that the games that are build with that pattern have the same problems with those functions.
